### DISCUSSION TITLE
Pair to Mac for iOS development?

### DISCUSSION THREAD

**Discussion by ward-horsfall** [codemagic-ci-cd/codemagic-docs#2426] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Other - please mention in your question

### Steps to reproduce

Uno Platform creating a new project and trying to set up a debugging session.

### Expected results

Allow iOS debbuging

### Actual results

Not sure how to do this (if product offers it).

**Comment by himesh-cm** [comment, marked as answer]
hi @ward-horsfall,
Debugging can be done on a local Mac machine, and you can use Codemagic for QA, building, and releasing.
You cannot pair Mac machines on codemagic for debugging purposes.

**Comment by himesh-cm** [comment]
While I’m closing our discussion for now, if you have any other questions or suggestions, feel free to reply back or start a new discussion anytime. Have a nice day!
